crates/ironrdp-web/src/config_parser.rs

```diff
@@ -0,0 +1,30 @@
+use wasm_bindgen::prelude::wasm_bindgen;
+
+#[wasm_bindgen]
+pub struct RdpConfigParser(ironrdp_propertyset::PropertySet);
```

Comment:
suggestion: I think, I would prefer we name it `RdpFile`, so it’s clear we are dealing with the .RDP file format.

```rust
let rdp_file = RdpFile::load(rdp_file_config);
let full_address = rdp_file.get_str("full address");
```

Alternatively, `PropertySet` is also exactly what it is. Could be:

```rust
let properties = PropertySet::load_rdp_file(rdp_file_config);
```